carbonmark/components/ProjectCard/index.tsx

```diff
@@ -61,8 +62,8 @@ export const ProjectCard: FC<Props> = (props) => {
         </Text>
         <div className={styles.tags}>
           <Vintage vintage={project.vintage} />
-          {project?.methodologies?.length > 1 ? (
-            project.methodologies.map((methodology: Methodology, index) => (
+          {!isEmpty(project?.methodologies) ? (
```

Comment:
this is not the same comparison, the intent is to handle _multiple_ methodologies.

Reply:
Ah whoops.. very true.

Reply:
This is a

Reply:
Because `project.methodologies` can be nullish this is a case where I would like to destructure the object (props) further to avoid `((project?.methodologies ?? []).length)` but this goes against the styleguide.

What do you reckon?

Reply:
I would do this, on second glance it appears the original ternery operator is not even necessary
```jsx
// fallback because methodologies is nullable
const categories = props.project.methodologies?.map(m => m.category) || [];

return (
  <>
  {categories.map((c, i) => <Category key={i} category={c} />)}
  </>
)
```